pkgs/build-support/rust/build-rust-package/default.nix

```
@@ -45,7 +45,7 @@
 , buildFeatures ? [ ]
 , checkFeatures ? buildFeatures
 , useNextest ? false
-, auditable ? true
+, auditable ? buildPackages.cargo-auditable.meta.broken
```

Shouldn't this be the inverse?

```suggestion
, auditable ? !buildPackages.cargo-auditable.meta.broken
```

---

oh noo

thank you!